sites/docs/src/content/docs/guidelines/pipelines/recommendations/publication_credit.md

```diff
@@ -18,6 +18,8 @@ All nf-core pipeline publications and presentations should cite [The nf-core fra
 
 If members of the nf-core community have provided significant input to the creation or maintenance of a pipeline, please consider adding them as coauthors on the pipeline publication or presentation.
 
+We also strongly recommend transparency when working on a project where the pipeline plays a significant role. If you plan to write a paper about a pipeline, please share this information prior and during writing of the manuscript with the community in the pipeline specific channel on the nf-core Slack workspace, so that other major contributors can join and support your effort.
+
 If your pipeline publication or presentation is an extension of a community pipeline, please discuss authorship with the main developers.
```

Comment:
```suggestion
If your pipeline publication or presentation is an extension of a community pipeline, please discuss authorship with the main developers.

Based on [ICMJE's recommendations](https://www.icmje.org/recommendations/) for authorship, we developed 4 criteria:

- Substantial contributions to the conception or design of the pipeline; or the acquisition, analysis, or interpretation of data used to test/demo the pipeline; AND
- Drafting the work or reviewing it critically for important intellectual content; AND
- Final approval of the version to be published; AND
- Agreement to be accountable for all aspects of the work in ensuring that questions related to the accuracy or integrity of any part of the work are appropriately investigated and resolved.
```

Reply:
I found [this link](https://prevention.ucsf.edu/sites/prevention.ucsf.edu/files/2024-07/Authorship%20Guidelines%20for%20DPS%20Community%20Engaged%20Research%20FINAL%20Doc%20071624.pdf). It expands on the criteria a little. I find the clause about not granting authorship to anyone not involved in the manuscript drafting to be a little strict. I would drop the second bullet point.

Maybe this is a discussion for the next core team meeting? Get lots of eyes on it as this is a reasonably large decision.

Reply:
Didn't read but might also be worth a skim [this](https://osf.io/s6h58/download/)

Reply:
```suggestion
If your pipeline publication or presentation is an extension of a community pipeline, please discuss authorship with the main developers.

Based on [ICMJE's recommendations](https://www.icmje.org/recommendations/) for authorship, we suggest you could evaluate inclusion of contributors as a co-author based on the following 4 criteria:

    - Substantial contributions to the conception or design of the pipeline; or the acquisition, analysis, or interpretation of data used to test/demo the pipeline; AND
    - Drafting the work or reviewing it critically for important intellectual content; AND
    - Final approval of the version to be published; AND
    - Agreement to be accountable for all aspects of the work in ensuring that questions related to the accuracy or integrity of any part of the work are appropriately investigated and resolved.
```

However I still don't _really_ like this, it's being too proscriptive and I worry about might put people off contributing pipelines to nf-core if they (PIs) are worried about the risk of getting into protracted arguments about authorships or feel that it's no-longer their pipeline and they include the publication with their thesis etc.

Furthmore, it could be someone wants authorship to recognise their contribution, but they don't care about the drafting the actual publication (it's not their job), so the 'AND' is going to make that difficult. Furthermore, someone may not want to be accountable for _all_ aspects of the work if it's some other lab who has done lab work analysis etc. in the publication.

To be clear: conceptually it is nice, but I don't feel we can control this.

We already have problems tracking all developers, if we now have to chase them to make sure they are following

Reply:
That's a good point. I guess it all comes back to what constitutes a substantial contribution and how to encourage thoughtful consideration for authorship in different scenarios. Maybe we could give some examples.

Something something something... substantial contribution. For example:
- Maintenance of xxxx 
- Development of xxxx
- Or other xxxx

Reply:
Yeah, that would work with examples. But ensure the wording is that they are just suggestions to help you decide

Reply:
@jfy133 / @mribeirodantas - shall we apply these suggested changes and then do a final review?